src/features/pages/deriv-ctrader/copy-trading/data.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { TradingPlatformCardDataType } from 'features/components/templates/trade-platform/types'
+import Image from 'features/components/atoms/image'
+import MultipleAssets from 'images/svg/trading-platforms/ctrader/multiple-assets.svg'
+import CallTheShots from 'images/svg/trading-platforms/ctrader/call-the-shots.svg'
+import RiskManagement from 'images/svg/trading-platforms/ctrader/risk-management.svg'
+import { localize } from 'components/localization'
+
+export const why_copy_trade: TradingPlatformCardDataType[] = [
```

Comment:
Pls confirm, that this is the notation for declaring constants.
In my experience, usually, it is WhyCopyTrade or WHY_COPY_TRADE.

Reply:
Hi @dan-deriv ,we are using the `snake_case` throughout the codebase for the const data